### ISSUE TITLE
logEvent doesn't work when called without the metadata parameter

### ISSUE THREAD

**Issue by nightaul** [intercom/intercom-cordova#11]
Hi,

When calling ```intercom.logEvent('event_name');``` no event gets sent to Intercom and the following gets logged in the console:
```
-[NSNull count]: unrecognized selector sent to instance 0x38ca53c0

*** WebKit discarded an uncaught exception in the webView:decidePolicyForNavigationAction:request:frame:decisionListener: delegate: <NSInvalidArgumentException> -[NSNull count]: unrecognized selector sent to instance 0x38ca53c0
```

Calling ```intercom.logEvent('event_name', {});``` or ```intercom.logEvent('event_name', {param: 'data'});``` works fine.

**Comment by jtreanor** [comment]
Thanks @nightaul. I'll fix this in our next release.

It should be a simple null check [here](https://github.com/intercom/intercom-cordova/blob/master/src/ios/IntercomBridge.m#L76).

**Comment by nightaul** [comment]
Great, thanks!

**Comment by jtreanor** [comment]
No problem, I have fixed it on 92091fd7cc98455c6e9421eabd80079caaa574cc.

This will be included in our next point release through npm.

**Comment by jtreanor** [comment]
This has been fixed in 1.0.6 :smile:
